js/src/personal/riyuan/components/about/About.tsx

```diff
@@ -1,28 +1,94 @@
-import { Text, Title } from '@mantine/core'
+import {
+  SimpleGrid,
+  Title,
+  Text,
+  List,
+  Image,
+  ListItem,
+  Flex,
+} from '@mantine/core'
+import { useMediaQuery } from '@mantine/hooks'
 import styles from '@/personal/riyuan/Riyuan.module.css'
+import aboutStyles from '@/personal/riyuan/components/about/About.module.css'
 
-export function About() {
+// Define an interface for the props
+interface InfoSectionProps {
```

Comment:
Use Type instead of interface for type definition. (If they work)

Reply:
Can you update here how you resolved this? Did `type InfoSectionProps` not work? What's the error?